# HG changeset patch
# User Thorsten Meyer <thorsten.meyier@gmx.de>
# Date 1264622253 -3600
#      Wed Jan 27 20:57:33 2010 +0100
# Node ID 1ca904d74f78de5d191562cfd8092a925890b1a5
# Parent  752f349052a2fc14ec33f99727fdf4452c48f585
Add emf to list of enhanced terminals for the gnuplot backend

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-01-27  Thorsten Meyer  <thorsten.meyier@gmx.de>
+
+	* plot/gnuplot_drawnow.m: Add "emf" to the list of enhanced
+	terminals for the gnuplot backend.
+
 2010-01-26  John W. Eaton  <jwe@octave.org>
 
 	* general/chop.m: New function.
 
 2010-01-26  Jaroslav Hajek  <highegg@gmail.com>
 
 	* optimization/fsolve.m: Support TypicalX, autoscale only if
 	AutoScaling is on, off by default. Fix default tolerances.
diff --git a/scripts/plot/gnuplot_drawnow.m b/scripts/plot/gnuplot_drawnow.m
--- a/scripts/plot/gnuplot_drawnow.m
+++ b/scripts/plot/gnuplot_drawnow.m
@@ -386,17 +386,18 @@ function [term, opts] = gnuplot_trim_ter
 endfunction
 
 function have_enhanced = gnuplot_is_enhanced_term (term)
   persistent enhanced_terminals;
   if (isempty (enhanced_terminals))
     ## Don't include pstex, pslatex or epslatex here as the TeX commands
     ## should not be interpreted in that case.
     enhanced_terminals = {"aqua", "dumb", "png", "jpeg", "gif", "pm", ...
-                          "windows", "wxt", "svg", "postscript", "x11", "pdf"};
+                          "windows", "wxt", "svg", "postscript", "x11", ...
+                          "pdf", "emf"};
   endif
   if (nargin < 1)
     ## Determine the default gnuplot terminal.
     term = gnuplot_default_term ();
   endif
   have_enhanced = any (strncmp (enhanced_terminals, term, min (numel (term), 3)));
 endfunction
 
